build/launcher-script.sh

```diff
@@ -3,6 +3,49 @@
 # Uncomment for debugging purposes:
 # set -eux
```

Comment:
I think we should always have `set -eux`, not just for debugging.

Reply:
In our case, we can't because the definition of `UNPRIVILEGED_USERNS_ENABLED` relies on a command call that will error out if it isn't enabled.